<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelConfig</class>
 <widget class="QDialog" name="ChannelConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="0" column="1" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_channel_1">
      <attribute name="title">
       <string>Canal 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="lb_ch1_model">
         <property name="text">
          <string>Nro de Modelo del Dispositivo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_ch1_model">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="pte_ch1_description">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_ch1_description">
         <property name="text">
          <string>Descripción</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chx_channel_1">
         <property name="text">
          <string>Habilitado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_channel_2">
      <attribute name="title">
       <string>Canal 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="lb_ch2_description">
         <property name="text">
          <string>Descripción</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_ch2_model">
         <property name="text">
          <string>Nro de Modelo del Dispositivo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_ch2_model">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="pte_ch2_description"/>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chx_channel_2">
         <property name="text">
          <string>Habilitado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_channel_3">
      <attribute name="title">
       <string>Canal 3</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QLabel" name="lb_ch3_model">
         <property name="text">
          <string>Nro de Modelo del Dispositivo</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_ch3_description">
         <property name="text">
          <string>Descripción</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_ch3_model">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="pte_ch3_description"/>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chx_channel_3">
         <property name="text">
          <string>Habilitado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_channel_4">
      <attribute name="title">
       <string>Canal 4</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QLabel" name="lb_ch4_model">
         <property name="text">
          <string>Nro de Modelo del Dispositivo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_ch4_model">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_ch4_description">
         <property name="text">
          <string>Descripción</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="pte_ch4_description"/>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chx_channel_4">
         <property name="text">
          <string>Habilitado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_channel_5">
      <attribute name="title">
       <string>Canal 5</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="pte_ch5_description"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_ch5_description">
         <property name="text">
          <string>Descripción</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_ch5_model">
         <property name="text">
          <string>Nro de Modelo del Dispositivo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_ch5_model">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chx_channel_5">
         <property name="text">
          <string>Habilitado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_channel_6">
      <attribute name="title">
       <string>Canal 6</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_ch6_model">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_ch6_description">
         <property name="text">
          <string>Descripción</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_ch6_model">
         <property name="text">
          <string>Nro de Modelo del Dispositivo</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="pte_ch6_description"/>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chx_channel_6">
         <property name="text">
          <string>Habilitado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_channel_7">
      <attribute name="title">
       <string>Canal 7</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="2" column="0">
        <widget class="QLabel" name="lb_ch7_description">
         <property name="text">
          <string>Descripción</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_ch7_model">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_ch7_model">
         <property name="text">
          <string>Nro de Modelo del Dispositivo</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="pte_ch7_description"/>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chx_channel_7">
         <property name="text">
          <string>Habilitado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_channel_8">
      <attribute name="title">
       <string>Canal 8</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="1" column="0">
        <widget class="QLabel" name="lb_ch8_model">
         <property name="text">
          <string>Nro de Modelo del Dispositivo</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="pte_ch8_description"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_ch8_description">
         <property name="text">
          <string>Descripción</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_ch8_model">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chx_channel_8">
         <property name="text">
          <string>Habilitado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pb_accept">
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pb_cancel">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
